packages/mc-scripts/src/config/optimizations.ts

```diff
@@ -0,0 +1,40 @@
+// Dependencies to be split/grouped into separate chunks.
+// This is useful to reduce the main bundle size and have more
+// dedicated caching strategy for specific chunks.
+// https://webpack.js.org/plugins/split-chunks-plugin/
+// https://rollupjs.org/configuration-options/#output-manualchunks
+const manualChunks = {
+  'apollo-client': ['@apollo/client'],
+  'core-js-pure': ['core-js-pure'],
+  'commercetools-uikit-icons': ['@commercetools-uikit/icons'],
+  moment: ['moment', 'moment-timezone'],
+  react: [
+    'react',
+    'react-dom',
+    'react-router',
+    'react-router-dom',
+    'react-intl',
+    '@reduxjs/toolkit',
+    'redux',
+    'react-redux',
+    'redux-logger',
+    'redux-thunk',
+  ],
```

Comment:
@emmenko the only change I made is to lump redux and react together; since redux is fairly small this seems fine to do.